src/main/java/com/teamwizardry/librarianlib/features/kotlin/ExtMisc.kt

```diff
@@ -31,10 +31,17 @@ import net.minecraft.world.World
 import net.minecraft.world.chunk.Chunk
 import net.minecraftforge.common.capabilities.Capability
 import net.minecraftforge.common.capabilities.ICapabilityProvider
-import java.lang.reflect.Modifier
-import java.lang.reflect.ParameterizedType
-import java.lang.reflect.Type
+import java.lang.reflect.*
+import java.lang.reflect.Array
 import java.util.*
+import java.util.List
```

Comment:
```
 +import java.lang.reflect.Array
 +import java.util.List
```
No idea why idea absolutely wants to have them here, it keeps adding them back -- I use neither in the class, and it prevents me from importing `kotlin.Array` and `kotlin.collections.List` >.<